http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -144,10 +154,21 @@ object KubernetesServiceDbQueries {
           _ <- nodepoolQuery.updateStatus(createCluster.nodepoolId, NodepoolStatus.Provisioning)
```

Comment:
To check my understanding: is `createCluster.nodepoolId` the "dummy" nodepool, distinct from the `nodepoolId` passed to this method?

Reply:
Correct. I renamed it to `defaultNodepoolId` in another revision but must have stashed it. The optional nodepoolId is one for standalone nodepool creation if there are no pre-created ones